packages/core/lib/commands/test/run.js

```diff
@@ -68,43 +86,50 @@ module.exports = async function (options) {
     inputFile: file
   });
 
-  if (config.networks[config.network]) {
-    await Environment.detect(config);
-    const { temporaryDirectory } = await copyArtifactsToTempDir(config);
-    const numberOfFailures = await prepareConfigAndRunTests({
-      config,
-      files,
-      temporaryDirectory
-    });
-    return numberOfFailures;
-  } else {
-    const ipcOptions = { network: "test" };
-    const port = await require("get-port")();
+  const configuredNetwork = config.networks[config.network];
+  const testNetworkDefined = configuredNetwork && config.network === "test";
```

Comment:
This variable could still use a renaming like I suggested, I think.

Reply:
Can we use "testNetworkDefinedAndUsed" as the variable name, @haltman-at ? Also suggested by @cds-amal . As it is related to the "test" network, so I am also in support of that. Any other suggestions anyone @eggplantzzz , @gnidan , @fainashalts ?

Reply:
Sure, that sounds fine.

Reply:
Okay, I will change the it and push another commit. Thanks @haltman-at .